src/Microsoft.Diagnostics.DebugServices.Implementation/ThreadServiceFromDataReader.cs

```diff
@@ -25,11 +25,16 @@ public ThreadServiceFromDataReader(IServiceProvider services, IDataReader dataRe
             _threadReader = (IThreadReader)dataReader;
```

Comment:
Feels odd to not assert this accessibility.

Reply:
The TargetFromDataReader.cs checks this and throws an exception if IThreadReader doesn't exists.